includes/Classifai/Admin/templates/onboarding-step-four.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Step-1 template for ClassifAI Onboarding.
+ *
+ * @package ClassifAI
+ */
+
+use function Classifai\get_post_types_for_language_settings;
+
+$onboarding_options   = get_option( 'classifai_onboarding_options', array() );
+$enabled_features     = isset( $onboarding_options['enabled_features'] ) ? $onboarding_options['enabled_features'] : array();
+$configured_providers = isset( $onboarding_options['configured_providers'] ) ? $onboarding_options['configured_providers'] : array();
+?>
+<h1 class="classifai-setup-heading">
+	<?php esc_html_e( 'Welcome to ClassifAI', 'classifai' ); ?>
+</h1>
+<div class="classifai-spacer"></div>
+<div class="classifai-setup__content__row">
+	<div class="classifai-setup__content__row__column">
+		<div class="classifai-setup-image">
+			<img src="https://via.placeholder.com/334x334" alt="<?php esc_attr_e( 'ClassifAI Setup', 'classifai' ); ?>" />
+		</div>
+	</div>
+	<div class="classifai-setup__content__row__column">
+		<form method="POST" action="">
+			<div class="classifai-step1-content">
+			<h1 class="classifai-setup-title">
+				<?php esc_html_e( 'ClassifAI configured successfully!', 'classifai' ); ?>
+			</h1>
+			<?php
+			$divider = false;
+			if ( count( array_intersect( array( 'computer_vision', 'openai_chatgpt' ), $configured_providers ) ) > 1 && isset( $enabled_features['language'] ) ) {
+				?>
+				<div class="classifai-feature-box">
+					<div class="classifai-feature-box-title">
+						<?php esc_html_e( 'Language', 'classifai' ); ?>
+					</div>
+					<div class="classifai-features">
+						<ul>
+							<?php
+							$types    = get_post_types_for_language_settings();
+							$features = isset( $enabled_features['language']['classify'] ) ? $enabled_features['language']['classify'] : array();
+							foreach ( $types as $posttype ) {
+								if ( ! isset( $features[ $posttype->name ] ) ) {
+									continue;
+								}
+								?>
+								<li class="classifai-enable-feature">
+									<span class="dashicons dashicons-yes-alt"></span>
+									<label class="classifai-feature-text">
+									<?php
+										// translators: %s is the post type label.
+										printf( esc_html__( 'Automatically tag %s', 'classifai' ), esc_html( $posttype->label ) );
+									?>
+									</label>
+								</li>
+								<?php
+							}
+							if ( isset( $enabled_features['language']['excerpt_generation'] ) ) {
+								?>
+								<li class="classifai-enable-feature">
+									<span class="dashicons dashicons-yes-alt"></span>
+									<label class="classifai-feature-text">
+										<?php esc_html_e( 'Automatic excerpt generation', 'classifai' ); ?>
```

Comment:
Since we don't actually automatically generate excerpts (you have to click a button to generate it) I think we may want to change the wording here a bit:
```suggestion
										<?php esc_html_e( 'Excerpt generation', 'classifai' ); ?>
```

Reply:
Thanks for adding note for this.